pkg/controllers/provisioning/provisioner.go

```diff
@@ -328,7 +329,7 @@ func (p *Provisioner) launch(ctx context.Context, opts LaunchOptions, node *sche
 		return cheapestOfferingPrice(iOfferings, node.Requirements) < cheapestOfferingPrice(jOfferings, node.Requirements)
 	})
 
-	logging.FromContext(ctx).Infof("Launching %s", node)
+	logging.FromContext(ctx).With("node", node).Infof("Launching Node")
```

Comment:
Can we inject this at the top of this function since this function is scoped to the node?

Reply:
What does this output? I haven't looked at the `String()` method for the `*scheduler.Node` but it might be too long to be considered a structured value

Reply:
Yeah, it's better as a string format